From 2d51911c0c650c3d6c66a7661da9821fbc228809 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Sabri=20=C3=9Cnal?=
 <161761531+yakushabb@users.noreply.github.com>
Date: Mon, 11 Nov 2024 16:31:15 +0300
Subject: [PATCH] Update metainfo

- Add missing the developer name
- Add vcs-browser and bugtracker URLs
---
 dists/one.alynx.FlipClock.metainfo.xml | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/dists/one.alynx.FlipClock.metainfo.xml b/dists/one.alynx.FlipClock.metainfo.xml
index 4c97181..15bca37 100644
--- a/dists/one.alynx.FlipClock.metainfo.xml
+++ b/dists/one.alynx.FlipClock.metainfo.xml
@@ -12,6 +12,7 @@
   <content_rating type="oars-1.1" />
   
   <launchable type="desktop-id">one.alynx.FlipClock.desktop</launchable>
+  <developer_name>Alynx Zhou</developer_name>
   
   <supports>
     <control>pointing</control>
@@ -26,6 +27,8 @@
     </p>
   </description>
   <url type="homepage">https://flipclock.alynx.one/</url>
+  <url type="bugtracker">https://github.com/AlynxZhou/flipclock/issues</url>
+  <url type="vcs-browser">https://github.com/AlynxZhou/flipclock</url>  
   <screenshots>
     <screenshot type="default">
       <image>https://flipclock.alynx.one/images/screenshot-1.png</image>
